### PR TITLE
#22 filtering bar

### PR DESCRIPTION
**Pull request by edouardmisset**
Creating a filter bar to filter through the displayed movies

### CODE REVIEW

**Comment by edouardmisset** [comment]
Filters work & the style matches the rest of the app.

File: src/components/ConvertGenreIdToName.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+// Replace this hardcoded list with a API call to the following endpoint (using the API key)
+const genreListEndpoint =
+  'https://api.themoviedb.org/3/genre/movie/list?api_key=f22eb05a70b166bd4e2c1312e15d8e8b';
+// Replace this hardcoded list with a API call to the following endpoint (using the API key)
+const discoverMovieListEndpoint =
+  'https://api.themoviedb.org/3/discover/movie?api_key=f22eb05a70b166bd4e2c1312e15d8e8b';
```
Comment (ComicScrip) [inline src/components/ConvertGenreIdToName.js:7]: Please put the API key in a (non-versionned) `.env` and a dummy value in a (versionned)`.env.sample` file.
See https://create-react-app.dev/docs/adding-custom-environment-variables/

File: src/components/ConvertGenreIdToName.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    .then(({ data }) => data.genres)
+    .then((genres) =>
+      genres.reduce((acc, val) => ({ ...acc, [val.id]: val.name }), {})
+    );
+
+const movieList = getDiscoverMovies();
+console.log(movieList);
+
+/**
+ * @description Function to return a list of movies (or a single movie) with the genres converted to names
```
Comment (ComicScrip) [inline src/components/ConvertGenreIdToName.js:35]: Oh you tried JSDoc, that's nice :)

**Review by ComicScrip — CHANGES_REQUESTED** [review]

File: src/components/ConvertGenreIdToName.js (hunk 1 above)
Comment (edouardmisset) [reply src/components/ConvertGenreIdToName.js]: Fixed this :)

**Comment by edouardmisset** [comment]
@ComicScrip : Can you merge our PR please ?

File: .gitignore [hunk 3]
```diff
@@ -23,3 +23,5 @@ yarn-debug.log*
 yarn-error.log*
 
 .vscode
+.env
+.env.sample
```
Comment (ComicScrip) [inline .gitignore:27]: .env should not be versionned, but .env.sample should be versionned

**Review by ComicScrip — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (edouardmisset) [reply .gitignore:27]: ok

**Comment by edouardmisset** [comment]
Fixed ?

**Comment by edouardmisset** [comment]
Oups PR should be into dev not main. Do you know how to change that @ComicScrip  ?

**Pull request merged**